ocaml/xapi/xapi_vm_lifecycle.ml

```diff
@@ -168,6 +169,9 @@ let check_op_for_feature ~__context ~vmr ~vmgmr ~power_state ~op ~ref ~strict =
 			| `changing_VCPUs_live
 					when lack_feature "feature-vcpu-hotplug"
 						-> some_err Api_errors.vm_lacks_feature_vcpu_hotplug
+			| `suspend | `checkpoint | `pool_migrate | `migrate_send
```

Comment:
Do you think it's worth a comment to make explicit that (in accordance with the OCaml language specification) the `when` guard applies to all four preceding operations, not just the last one?